src/System.Sorting/tests/Program.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
```

Comment:
What's the point of this file?  Was it to make things easier to test while you were developing?  I think we should remove it since in general we just use XUnit to run the tests.